tox.ini

```diff
@@ -19,3 +19,28 @@ deps =
     yamllint==1.15.0
 commands =
     bash -c "yamllint eve/main.yml salt/**/*.yaml"
+
+[testenv:ci-tests]
+deps =
+    pytest
+    pytest_bdd
+    testinfra
+    paramiko
```

Comment:
Does this need to be an explicit dependency?

Reply:
paramiko is a backend of testinfra but it can live without it (by using other backends like ansible or salt etc) I guess I had to install it but will see about it when moving deps to a requirement file